File: Application/SkuciSe/app/src/main/java/com/example/skucise/models/HelperModels/ReservationMessage.kt
package com.example.skucise.models.HelperModels

import org.joda.time.DateTime
import java.io.Serializable

data class ReservationMessage(
    val senderId: Long?,
    val receiverId: Long?,
    val advertId: Long?,
    val title: String,
    val dateOfReservation: String? = null,
    val flag: Int,
    val answer: Int? = null
) : Serializable {
}

File: Application/Backend/SkuciSeAPI/SkuciSeAPI/Models/HelperModels/ReservationMessage.cs
using FirebaseAdmin.Messaging;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkuciSeAPI.Models.HelperModels
{
    public class ReservationMessage
    {
        public long SenderId { get; set; }
        public long ReceiverId { get; set; }
        public long AdvertId { get; set; }
        public string Title { get; set; }
        public DateTime DateOfReservation { get; set; }
        public Message Message { get; set; }
        public Flag Flag { get; set; }
        public Status Answer { get; set; }
    }
}
